src/measure_theory/measure/vector_measure.lean

```diff
@@ -985,6 +985,81 @@ end
 
 end absolutely_continuous
 
+/-- Two vector measures `v` and `w` are said to be mutually singular if there exists a measurable
+set `s`, such that for all `t ⊆ s`, `v t = 0` and for all `t ⊆ sᶜ`, `w t = 0`.
+
+We note that we do not require the measurability of `t` in the definition since this makes it easier
+to use. This is equivalent to the definition which requires measurability. To prove
+`mutually_singular` with the measurability condition, use
+`measure_theory.vector_measure.mutually_singular.mk`. -/
+def mutually_singular (v : vector_measure α M) (w : vector_measure α N) : Prop :=
+∃ (s : set α), measurable_set s ∧ (∀ t ⊆ s, v t = 0) ∧ (∀ t ⊆ sᶜ, w t = 0)
+
+localized "infix ` ⊥ᵥ `:60 := measure_theory.vector_measure.mutually_singular" in measure_theory
+
+namespace mutually_singular
+
+variables {v v₁ v₂ : vector_measure α M} {w w₁ w₂ : vector_measure α N}
+
+lemma mk (s : set α) (hs : measurable_set s)
+  (h₁ : ∀ t ⊆ s, measurable_set t → v t = 0)
+  (h₂ : ∀ t ⊆ sᶜ, measurable_set t → w t = 0) : v ⊥ᵥ w :=
+begin
+  refine ⟨s, hs, λ t hst, _, λ t hst, _⟩;
+  by_cases ht : measurable_set t,
+  { exact h₁ t hst ht },
+  { exact not_measurable v ht },
+  { exact h₂ t hst ht },
+  { exact not_measurable w ht }
+end
+
+lemma symm (h : v ⊥ᵥ w) : w ⊥ᵥ v :=
+let ⟨s, hmeas, hs₁, hs₂⟩ := h in
+  ⟨sᶜ, hmeas.compl, hs₂, λ t ht, hs₁ _ (compl_compl s ▸ ht : t ⊆ s)⟩
+
+lemma zero_right : v ⊥ᵥ (0 : vector_measure α N) :=
+⟨∅, measurable_set.empty, λ t ht, (subset_empty_iff.1 ht).symm ▸ v.empty, λ _ _, zero_apply _⟩
+
+lemma zero_left : (0 : vector_measure α M) ⊥ᵥ w :=
+zero_right.symm
+
+lemma add_left [t2_space N] [has_continuous_add M] (h₁ : v₁ ⊥ᵥ w) (h₂ : v₂ ⊥ᵥ w) : v₁ + v₂ ⊥ᵥ w :=
+begin
```

Comment:
Can you prove this more easily by arguing via total variation? `v1.tv` and `v2.tv` are orthogonal to `w.tv`, and hence so is `v1.tv + v2.tv`. Then `(v1 + v2).tv <= v1.tv + v2.tv`, and some previous monotonicity lemma finishes?

(I don't know this part of the library at all, nor this maths very well, so perhaps I'm on the wrong track.)

Reply:
I think that will be less general since total variation requires `N` and `M` to be normable

Reply:
Ah, okay! I hadn't paid enough attention to your typeclass hypotheses.